setup.ps1

```diff
@@ -11,6 +12,8 @@ $connectionPort = 1521
 $runnerOs = $Env:RUNNER_OS ?? "Linux"
 $resourceGroup = "GitHubActions-RG"
 $healthCheckCommand = ""
+$listenerHealthCheckCommand = ""
```

Comment:
Is "listener" the best term to use here? As I understand it, this check ensures we can run SQL against the DB (the init script), so maybe:

```suggestion
$sqlHealthCheckCommand = ""
```